packages/@glimmer/opcode-compiler/lib/opcode-builder/helpers/vm.ts

```diff
@@ -79,13 +79,13 @@ export function CallDynamic(
   named: WireFormat.Core.Hash,
   append?: () => void
 ): void {
-  op(Op.Load, $v0);
   op(MachineOp.PushFrame);
   SimpleArgs(op, positional, named, false);
-  op(Op.DynamicHelper, $v0);
+  op(Op.Dup, $fp, 1);
```

Comment:
is there a reference somewhere to see what each opcode's purpose is?

Reply:
I think https://github.com/glimmerjs/glimmer-vm/blob/master/packages/@glimmer/vm/lib/opcodes.toml is probably the best place.